pkgs/top-level/julia-packages.nix

```diff
@@ -0,0 +1,60 @@
+{ pkgs
+, lib
+, stdenv
+, newScope
+, julia
+}:
+
+with lib;
+
+makeScope newScope (self:
+  let
+    callPackage = self.callPackage;
+
+    buildJuliaPackage = callPackage ../development/compilers/julia/build-julia-package.nix {
+      inherit lib stdenv;
```

Comment:
```suggestion
```
Those should be added by callPackage automatically

Reply:
I'll change that.